app/src/org/commcare/activities/connect/ConnectIdBiometricConfigActivity.java

```diff
@@ -24,22 +31,27 @@ public class ConnectIdBiometricConfigActivity extends CommCareActivity<ConnectId
     private ConnectIdBiometricConfigActivityUiController uiController;
     private BiometricManager biometricManager;
 
+    private BiometricPrompt.AuthenticationCallback biometricPromptCallbacks;
+
+    private boolean attemptingFingerprint = false;
+    private boolean allowPassword = false;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setTitle(getString(R.string.connect_verify_title));
         biometricManager = BiometricManager.from(this);
-
+        biometricPromptCallbacks = preparePromptCallbacks();
         uiController.setupUI();
-
+        allowPassword = getIntent().hasExtra(ConnectConstants.ALLOW_PASSWORD) && getIntent().getStringExtra(ConnectConstants.ALLOW_PASSWORD).equals("true");;
```

Comment:
This is no longer an option since recovery passwords are deprecated, we can get rid of allowPassword and any related functionality.